internal/home/home.go

```diff
@@ -887,3 +888,16 @@ type jsonError struct {
 	// Message is the error message, an opaque string.
 	Message string `json:"message"`
 }
+
+// getTLSCiphers check for overriden tls ciphers, if the slice is
+// empty, then default safe ciphers are used
+func getTLSCiphers() []uint16 {
+	var cipher []uint16
```

Comment:
We prefer to use named returns in such cases.  See the [guidelines item](https://github.com/AdguardTeam/CodeGuidelines/blob/master/Go/Go.md#li-da463e1c).